xgboost-operator.yaml

```diff
@@ -0,0 +1,201 @@
+#--------------------------------------------------
+#- Single configuration deployment YAML for xgboost-Operator
+#- Include:
+#      CRD deployment
+#      Namespace deployment
+#      RBAC deployment
+#      xgboost-operator container deployment set     
+# --------------------------------------------------
+---
```

Comment:
this is not namespace based ? I do not see the namespace configure

Reply:
namespace configure is at line 45

Reply:
This project is generated by kubebuilder and user supposed to use `make` to deploy the manifests. 
Another option is to use `kustomize` to build the manifest and generate one yaml file. 

`my-namepsace` doesn't sounds like a reasonable name. It's better to use `xgboost-operator` or `kubeflow`

Reply:
Thank you so much for your advice. kustomize sounds like a better option

Reply:
in the production, we do not have to use the kubebuild, we just need to use the yaml to deploy the controller. Then the namespace would be more meaningful, for example, the namespace might be related to user group information, then xgboost-operator can be the name for the namespace.